modules/advanced-seo/advanced-seo-posts.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Class containing utility static methods for managing SEO custom descriptions for Posts and Pages.
+ */
+class Advanced_SEO_Posts {
```

Comment:
We may want to namespace these classes (`Jetpack_SEO_XXXX`) to avoid potential conflicts.